.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Description
+<!--
+Please add an informative description that covers the changes made by the pull request.
+
+If applicable, reference the bug/issue that this pull request fixes here.
+-->
+
+This checklist is used to make sure that common guidelines for a pull request are followed.
+
+- [ ] **I have read the [contribution guidelines](./CONTRIBUTING.md).**
+- [ ] Title of the pull request is clear and informative.
+- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR.
+- [ ] The pull request does not introduce breaking changes (unless a major version change occurs in the assembly and module).
+- [ ] If applicable, the public code is properly documented.
```

Comment:
XML documentation is enabled on the main projects, but this wording is in place to let people know that it is a requirement of the PR.